package.json

```diff
@@ -23,6 +23,7 @@
     "react-dom": "17.0.1",
     "react-hook-form": "^6.15.5",
     "sass": "^1.30.0",
+    "url": "^0.11.0",
```

Comment:
Q: What does this give us, that's not already there in the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL)?

Reply:
i read about this but i thought those two are same.
changing it